frontend/src/components/applications/CancelApplicationDialog.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <AppDialog v-model="isDisplayed" title="Confirm" :isLoading="isLoading" persistent max-width="50%" @close="closeDialog">
+    <template #content>
+      <div align="center" class="confirm-dialog-text">Are you sure you want to cancel this application?</div>
+    </template>
+    <template #button>
+      <v-row class="mt-2" justify="space-around">
```

Comment:
We should add v-cols to align better at low resolution. See what was done here https://github.com/bcgov/ecc-ofm/blob/ofmcc-729-update-user/frontend/src/components/settings/ManageUserDialog.vue.

Reply:
it is fixed